src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -16,373 +16,490 @@ public static class Sse
 
         /// <summary>
-        /// __m128 _mm_cvtsi32_ss (__m128 a, int b)
+        /// __m128 _mm_cvtsi32_ss (__m128 a, int b);
+        ///   CVTSI2SS xmm, reg/m32
         /// </summary>
         public static Vector128<float> ConvertToVector128SingleScalar(Vector128<float> upper, int value) => ConvertToVector128SingleScalar(upper, value);
         /// <summary>
-        /// __m128 _mm_cvtsi64_ss (__m128 a, __int64 b)
+        /// __m128 _mm_cvtsi64_ss (__m128 a, __int64 b);
+        ///   CVTSI2SS xmm, reg/m64
         /// </summary>
         public static Vector128<float> ConvertToVector128SingleScalar(Vector128<float> upper, long value) => ConvertToVector128SingleScalar(upper, value);
 
         /// <summary>
-        /// int _mm_cvttss_si32 (__m128 a)
+        /// int _mm_cvttss_si32 (__m128 a);
+        ///   CVTTSS2SI r32, xmm/m32
         /// </summary>
         public static int ConvertToInt32WithTruncation(Vector128<float> value) => ConvertToInt32WithTruncation(value);
         /// <summary>
-        /// __int64 _mm_cvttss_si64 (__m128 a)
+        /// __int64 _mm_cvttss_si64 (__m128 a);
+        ///   CVTTSS2SI r64, xmm/m32
         /// </summary>
         public static long ConvertToInt64WithTruncation(Vector128<float> value) => ConvertToInt64WithTruncation(value);
 
         /// <summary>
-        /// __m128 _mm_div_ps (__m128 a,  __m128 b)
+        /// __m128 _mm_div_ps (__m128 a,  __m128 b);
+        ///   DIVPS xmm, xmm/m128
         /// </summary>
         public static Vector128<float> Divide(Vector128<float> left, Vector128<float> right) => Divide(left, right);
 
         /// <summary>
-        /// __m128 _mm_div_ss (__m128 a,  __m128 b)
+        /// __m128 _mm_div_ss (__m128 a,  __m128 b);
+        ///   DIVSS xmm, xmm/m32
         /// </summary>
         public static Vector128<float> DivideScalar(Vector128<float> left, Vector128<float> right) => DivideScalar(left, right);
 
         /// <summary>
-        /// __m128 _mm_loadu_ps (float const* mem_address)
+        /// __m128 _mm_loadu_ps (float const* mem_address);
+        ///   MOVUPS xmm, m128
         /// </summary>
         public static unsafe Vector128<float> LoadVector128(float* address) => LoadVector128(address);
 
         /// <summary>
-        /// __m128 _mm_load_ss (float const* mem_address)
+        /// __m128 _mm_load_ss (float const* mem_address);
+        ///   MOVSS xmm, m32
         /// </summary>
-        public static unsafe Vector128<float> LoadScalar(float* address) => LoadScalar(address);
+        public static unsafe Vector128<float> LoadScalarVector128(float* address) => LoadScalarVector128(address);
 
         /// <summary>
-        /// __m128 _mm_load_ps (float const* mem_address)
+        /// __m128 _mm_load_ps (float const* mem_address);
+        ///   MOVAPS xmm, m128
         /// </summary>
         public static unsafe Vector128<float> LoadAlignedVector128(float* address) => LoadAlignedVector128(address);
```

Comment:
Perhaps the naming could follow the scheme  `action` `on whom` `how` -> `LoadVector128Aligned`

Reply:
See my comment here: https://github.com/dotnet/coreclr/pull/15923#issuecomment-361456561

I don't think `Aligned` is the how, rather I think `Aligned Vector` is the type.